integr8ly/keycloak-client
# MGDAPI-6044 Bump golang.org/x/net bump to v0.17.0

# Issue Link

Jira: https://issues.redhat.com/browse/MGDAPI-6060

# What

New KC Client rev to 0.1.14

# Verification Steps

`checkout branch`

Run `make build`

Run  `make test/unit`

Ensure that there are no errors
